### DISCUSSION TITLE
Lighthouse Summary shows wrong device emulation information

### DISCUSSION THREAD

**Discussion by chatur-sriganesh** [GoogleChrome/lighthouse#13434] [Q&A]
Hi,
Tried running lighthouse from puppeteer script with device emulation as "puppeteer.devices['iPad landscape']". Also visually i can see that resolution is now changed to that of an ipad.
However the report still shows the following:

"Emulated Moto G4
Slow 4G throttling
4x slowdown"

Any idea if i'm missing something here?

**code snippet i am using:**

const iPad = puppeteer.devices['iPad landscape'];
  const browser = await puppeteer.launch({headless: false});
  const page = await browser.newPage();
  const flow = await startFlow(page, {name: 'Squoosh snapshots'});
  await page.emulate(iPad);
  await page.goto('https://squoosh.app/', {waitUntil: 'networkidle0'});
  await page.emulate(iPad);

**Comment by adamraine** [comment]
By default, Lighthouse will use its own emulation settings regardless of what you set up with puppeteer beforehand. I would recommend configuring your emulation settings in the config. See our [config docs](https://github.com/GoogleChrome/lighthouse/blob/master/docs/configuration.md).

That being said, we don't offer much range for the form factor string. "Emulated Moto G4" is always used if mobile emulation is configured, regardless of other emulation settings. Looks like https://github.com/GoogleChrome/lighthouse/issues/7053 is tracking this.
